client/src/Components/Post.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import Title from './Title'
+import Content from './Content'
+import Img from './Img'
+
+export default function Post({posts}) {
```

Comment:
I would be careful with your naming here. I would name the prop `post` (singular) not the plural. This component only takes *one* post, not many. So name your props accordingly

Reply:
I'll pay more attention next time!